lib/apiservers/engine/backends/container.go

```diff
@@ -50,6 +50,7 @@ import (
 
 	"github.com/vishvananda/netlink"
 
+	"encoding/json"
```

Comment:
we do; I lazily allowed emacs to add this import. Usually `goimports` complains at me if something got put in the wrong place but it missed this one.